### ISSUE TITLE
about a package

### ISSUE THREAD

**Issue by star-study** [maybeliuchuan/Task-Oriented-Semantic-Communications#3]
Hello!
I want to ask a question about the misc_functions in the ResNet18_CAM.py. Is the misc_functions a costume package you write but not included in these files or a official package?
`from misc_functions import save_class_activation_images`

**Comment by maybeliuchuan** [comment]
Yes, this file is used to visualize the activation map. However, we do not use this and we only need to obtain the weights before visualization.
